### ISSUE TITLE
Leverage SystemD

### ISSUE THREAD

**Issue by JohnMertz** [MailCleaner/MailCleaner-Next#17]
Local init scripts (/usr/mailcleaner/etc/init.d/<service>) should be a wrapper which includes necessary 'dump' steps, before calling system init script via SystemD for actual process management.

- [x] Convert existing start/stop logic to SystemD units
  - [x] apache
  - [x] clamd
  - [x] clamspamd
  - [ ] ~dccifd~
  - [x] exim_stage1
  - [x] exim_stage2
  - [x] exim_stage4
  - [ ] fail2ban
  - [x] firewall
  - [x] greylistd
  - [x] mailcleaner
  - [x] mailscanner
  - [x] mariadb_master
  - [x] mariadb_slave
  - [x] newsld
  - [x] ntpd
  - [x] preftdaemon
  - [x] snmpd
  - [x] spamd
  - [x] spamhandler
  - [x] spamloggerdaemon
  - [x] spamloggerdaemon
  - [x] statsdaemon
- [x] Migrate 'get_status.pl'

**Comment by JohnMertz** [comment]
MariaDB no longer appears to block the restart. However, incomplete service `dccifd` blocks `start` because of permission issues and failure to open non-existent files. I need to just remove it as a blocking dependency.

**Comment by JohnMertz** [comment]
`dccifd` issue noted in another Issue. `fail2ban` also has an issue. Continue work in those more specific issues.

**Comment by JohnMertz** [comment]
Services don't completely start correctly during initial start-up. This is because various services are trying to connect to mariadb@slave too quickly. The `After=mariadb@slave.service` option does not seem to be sufficient, nor is setting a delay after the slave services with `ExecStartPost=sleep 1`, since the `ExecStartPre=<dumper script>` which is trying to connect does not wait on that.

Need to find a proper way of enforcing a delay. Perhaps add a dependency on the mariadb socket instead of service...
